packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/index.css

```diff
@@ -22,21 +22,53 @@ limitations under the License.
   line-height: 17px;
   font-size: 13px;
   font-family: monospace;
+
   white-space: pre-wrap;
+  white-space: -moz-pre-wrap;
+  white-space: -pre-wrap;
+  white-space: -o-pre-wrap;
```

Comment:
how does this work? You're redefining the same property, isn't it going to only keep the latest value?

Reply:
The `react-json-view-lite` library had a default styling in their [code](https://github.com/AnyRoad/react-json-view-lite/blob/release/src/styles.module.css#L3)

I copied the base CSS from there and then overridden some properties based on our design

Reply:
let's remove it. I don't know what it means. Vendor prefixes are usually used with attribute names, I've never seen them used in values.

Reply:
Sure